content/md/products-super-power/how-to-use-the-product-grid.md

```diff
@@ -55,6 +55,27 @@ The available channels are those concerned with the locale selected higher up on
 Like the locale, the channel selected on the product grid’s page is applied when editing a product form.
 :::
 
+# Product grid display (_since 2.1_)
+
+To highlight the products' images in the product grid, a new `Gallery` display of the product grid is now available. This display can be used to quickly find products by their images and update them.
+
+With this new display, the following properties are displayed in the grid (you cannot choose them, there is no `Columns` button available):
+- Identifier for a product (attribute defined as product identifier) or code for a [product model](/articles/what-about-products-variants.html#what-is-a-product-model)
+- Label: attribute used as "label" in the [product's family](/articles/manage-your-families.html#create-a-family)
+- Image: attribute used as "main picture" in the [product's family](/articles/manage-your-families.html#create-a-family)
+- Completeness for a product
+- Number of complete variant products out of the number of variant products for a [product model](/articles/what-about-products-variants.html#what-is-a-product-model) (ex: 2/3 means that 2 variants out of 3 for the product model are complete)
+
+Other grid features also apply to this new display (channel, locale and categories selection, filters, pagination, products selection, etc.).
+1.  Click on `List` to change the display  
```

Comment:
A sentence to introduce the steps, like "To use the "Gallery" display:"

Reply:
yes